packages/valory/skills/twitter_scoring_abci/behaviours.py

```diff
@@ -79,6 +81,34 @@ def params(self) -> Params:
         return cast(Params, super().params)
 
 
+class OpenAICallCheckBehaviour(TwitterScoringBaseBehaviour):
+    """TwitterCollectionBehaviour"""
+
+    matching_round: Type[AbstractRound] = OpenAICallCheckRound
+
+    def async_act(self) -> Generator:
+        """Do the act, supporting asynchronous execution."""
+        with self.context.benchmark_tool.measure(self.behaviour_id).local():
+            current_time = cast(
+                SharedState, self.context.state
+            ).round_sequence.last_round_transition_timestamp.timestamp()
+            # Reset the window if the window expired before checking
+            self.params.openai_calls.reset(current_time=current_time)
+            if self.params.openai_calls.max_tweets_reached():
+                content = OpenAICallCheckRound.API_CALL_EXCEEDED
+            else:
+                content = OpenAICallCheckRound.API_CALL_REMAINING
```

Comment:
Why not set the content to `None`? That way you would not need to override the round's `end_block` at all.